MapboxNavigation/NavigationMapView.swift

```diff
@@ -126,6 +126,13 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     var altitude: CLLocationDistance
     var routes: [Route]?
     var isAnimatingToOverheadMode = false
+    var primaryRoutePoints: RoutePoints?
+    var primaryRouteLineGranularDistances: RouteLineGranularDistances?
+    var primaryRouteRemainingDistancesIndex: Int?
+    var routeLineTracksTraversal: Bool = false
+    var fractionTraveled: Double = 0.0
+    var preFractionTraveled: Double = 0.0
+    var timer: Timer? = nil
```

Comment:
What about describing purpose of this timer? Feel free to add another name if you feel it describes it better.
```suggestion
    var vanishingRouteLineUpdateTimer: Timer? = nil
```

Reply:
Thanks for the suggestion, I'll fix it